components/channel_header/index.js

```diff
@@ -73,6 +74,8 @@ function makeMapStateToProps() {
             isMuted: isCurrentChannelMuted(state),
             isQuickSwitcherOpen: isModalOpen(state, ModalIdentifiers.QUICK_SWITCH),
             hasGuests: stats.guest_count > 0,
+            getDisplayNameByUserId: (userId) => getDisplayNameByUserId(userId, state),
+            handleFormattedTextClick: (e) => handleFormattedTextClick(e, getCurrentRelativeTeamUrl(state))
```

Comment:
And same thing here as well. This one is easier to handle since we just need to pass the current relative team URL into the `ChannelHeader` as a prop instead of passing a function. Alternatively, you could see if you could make `handleFormattedTextClick` into a redux action like I mentioned above.

Reply:
whoops... must've been tired when doind that one